src/Views/PointBlueViews.php

```diff
@@ -6,13 +6,17 @@
 
 class PointBlueViews extends Command
 {
-	const VIEWS_DEST_PATH = '/../../../../../resources/views/';
-	const PARTIALS_DEST_PATH = 'partials/';
-	const UNIVERSAL_PARTIALS_DEST_PATH = 'universal/';
-	const FOOTER_BLADE_DEST_FILENAME = 'pb-footer.blade.php';
+    const VIEWS_DEST_PATH = '/../../../../../resources/views/';
```

Comment:
these commands are always run in laravel, right?  we can use [resource_path()](https://laravel.com/docs/5.4/helpers#method-resource-path) instead of this.